src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Color color;
+    private final int number;
+
+    public Ball(Color color, int number) {

```

Comment:
according to the task
```suggestion
    private final String color;
    private final int number;

    public Ball(String color, int number) {
```